03_letters/willshen8/letters.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strconv"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func letters(s string) map[rune]int {
+	letterMap := make(map[rune]int)
+
+	for _, value := range s {
+		letterMap[value]++
+	}
+	return letterMap
+}
+
+func sortLetters(m map[rune]int) []string {
+	frequencySlice := []string{}
```

Comment:
Go style is to use 
`var frequencySlice []string{}` for initialising slices, 
or 
`frequencySlice := make([]string{}, 0, len(m))` for optimising for performance

Reply:
https://github.com/golang/go/wiki/CodeReviewComments

Reply:
Thanks, I'll take up the option 2 for better performance.